xAnalyzer/xanalyzer.cpp

```diff
@@ -1015,8 +1015,11 @@ void SetAutoCommentIfCommentIsEmpty(INSTRUCTIONSTACK *inst, char *CommentString,
 					}
 					else
 					{
-						strcat_s(CommentString, CommentStringCount, " => ");
-						strcat_s(CommentString, CommentStringCount, szComment);
+						if (strstr(szComment, CommentString) == nullptr)
```

Comment:
There's an option to avoid this in "Clear Previous Data/Auto Comments". The goal behind allowing this is in order for xAnalyzer to work with .map importers plugins seamlessly.